tests/e2e_logging.py

```diff
@@ -337,6 +339,88 @@ def test_view_history(network, args):
     return network
 
 
+class SentTxs:
+    # view -> seqno -> status
+    txs = defaultdict(lambda: defaultdict(lambda: TxStatus.Unknown))
+
+    @staticmethod
+    def update_status(view, seqno, status=None):
+        current_status = SentTxs.txs[view][seqno]
+        if status is None:
+            # If you don't know the current status, just ensure it is present but don't change it
```

Comment:
What does "ensure it is present" mean here?

Reply:
I've reworded the comment to hopefully be clearer. Essentially if you pass none, then the only action is accessing this `view.seqno` in the defaultdict (potentially creating it with the default `Unknown` status). I convinced myself that this was maybe necessary in case a previous call has already probed this Tx ID's status? Maybe we never know that it is currently 'Unknown'? It may be unneeded in the current test form.